### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by SamVerschueren**
@sindresorhus @kevva

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -12,92 +23,65 @@ const fixtures = [
...
+	const deletedFiles = await fn(['*.tmp', '!1*'], {cwd: t.context.tmp, dryRun: true});
+	exists(t, ['1.tmp', '2.tmp', '3.tmp', '4.tmp', '.dot.tmp']);
+	t.same(deletedFiles, [
+		path.join(t.context.tmp, '2.tmp'),
+		path.join(t.context.tmp, '3.tmp'),
+		path.join(t.context.tmp, '4.tmp')
+	]);
 });
 
-test.serial('options are optional', async () => {
```
Comment (SamVerschueren) [inline test.js]: Tested with the first test

File: test.js [hunk 2]
```diff
@@ -12,92 +23,65 @@ const fixtures = [
...
-test.afterEach(() => fixtures.forEach(fs.removeSync));
+test.beforeEach(t => {
+	process.chdir(cwd);
+	t.context.tmp = tempfile();
+	fixtures.forEach(fixture => fs.ensureFileSync(path.join(t.context.tmp, fixture)));
+});
 
-test.serial('delete files - async', async t => {
+test('delete files - async', async t => {
+	process.chdir(t.context.tmp);
```
Comment (SamVerschueren) [inline test.js]: Seemed like the only way to let this working (without options) together with line 27. If you don't care about testing it without options, we could remove all the `chdir` stuff.

**Comment by sindresorhus** [comment]
LGTM

**Comment by kevva** [comment]
Nice :+1:

File: test.js (hunk 2 above)
Comment (sindresorhus) [inline test.js]: I think it's fine. It makes the tests more robust.

**Comment by SamVerschueren** [comment]
Awesome. Thanks! :beers:

**Pull request merged**